Source/WebCore/platform/graphics/GraphicsContext.h

```diff
@@ -256,6 +256,8 @@ class GraphicsContext {
 
     virtual bool needsCachedNativeImageInvalidationWorkaround(RenderingMode) { return true; }
 
+    virtual bool needsNativeImageMemoryOwnership() const { return false; }
```

Comment:
This is a pretty obscure name. Can we just call it `preferredImageDecodingMode()`?